recipes/install.rb

```diff
@@ -556,6 +556,17 @@
   action :create_if_missing
 end
 
+cauth = File.basename(node['hopsworks']['cauth_url'])
+
+remote_file "#{theDomain}/lib/#{cauth}" do
+  node['install']['enterprise']['install'].casecmp?("true") ? headers( "Authorization"=>"Basic #{ Base64.encode64("#{node['install']['enterprise']['username']}:#{node['install']['enterprise']['password']}").gsub("\n", "") }" ) : headers({})
```

Comment:
Use this function https://github.com/logicalclocks/hops-hadoop-chef/blob/master/recipes/install.rb#L326 Also, set the sensitivity flag